stack.yaml

```diff
@@ -11,3 +11,6 @@ extra-deps:
 - bitvec-1.0.2.0
 - vector-algorithms-0.8.0.3
 - quiet-0.2
+- optics-core-0.3
```

Comment:
Why do we depend on `optics-core`? Is it a transitive dependency or do we use `optics-core`? If the latter, then we use `lens` in the compiler, so I'd prefer to use the same optics library on the language and the compiler sides.

Reply:
`optics-core` is the lightweight equivalent of `optics`, as `optics` depends on `optics-core`.  I prefer `optics-core` to `lens` as I find them easier to use and more lightweight, but there is nothing preventing us from using `lens`.